pkg/controller/plan/adapter/openstack/builder.go

```diff
@@ -420,42 +420,49 @@ func (r *Builder) mapResources(vm *model.Workload, object *cnv.VirtualMachineSpe
 	cpuCores = 1
 	cpuThreads = 1
 
-	if flavorCPUSockets, ok := vm.Flavor.ExtraSpecs[FlavorCpuSockets]; ok {
-		if sockets, err := strconv.Atoi(flavorCPUSockets); err == nil {
-			cpuSockets = sockets
+	if imageCPUSockets, ok := vm.Image.Properties[CpuSockets]; ok {
+		if socketsStr, ok := imageCPUSockets.(string); ok {
+			if sockets, err := strconv.Atoi(socketsStr); err == nil {
+				cpuSockets = sockets
+			}
```

Comment:
Is it really fine to ignore the error here? Should we perhaps at least log it?

Reply:
I guess there is some validation on the openstack side of these values but +1 for logging - it can help to debug misalignment between the original and the imported vms if it ever happens

Reply:
Done.